### PR TITLE
Rewrite/feature/detekt

### PR DESCRIPTION
**Pull request by patrickhoette**

### CODE REVIEW

File: Library/.idea/dictionaries/patrickhoette.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="patrickhoette">
```
Comment (thedokuijf) [inline Library/.idea/dictionaries/patrickhoette.xml:2]: Y you pushing yourself into the project? 😄 @patrickhoette

**Review by thedokuijf — COMMENTED** [review]
Library/.idea/dictionaries/patrickhoette.xml ?

File: Library/.idea/dictionaries/patrickhoette.xml (hunk 1 above)
Comment (patrickhoette) [reply Library/.idea/dictionaries/patrickhoette.xml:2]: nice catch. Let me look into that.

Comment (patrickhoette) [reply Library/.idea/dictionaries/patrickhoette.xml:2]: Its safes the dictionaries by users name... why? Its the project dictionary. Well back on the git ignore it goes.

**Pull request merged**